### ISSUE TITLE
H150i Fans Are Unresponsive

### ISSUE THREAD

**Issue by Meh7991** [EvanMulawski/FanControl.CorsairLink#108]
[log.zip](https://github.com/EvanMulawski/FanControl.CorsairLink/files/12830958/log.zip)

Not sure what has changed, but my front three fans don't respond to the curves again. I will try previous versions of this plugin to see if any of them work and will update this post. 

Attached are my logs, I can enable error logging if necessary?

**Comment by Meh7991** [comment]
Just tried a bunch of older releases but they're all the same. 

I'm running V171 of FanControl - all my fans were working perfectly when V171 was released a few days ago, it's just I booted my system earlier today and I could physically see the front fans spinning much slower than usual. 

Maybe there was some firmware update that released for the cooler and installed without my knowledge and that's messing the plugin up?
